rskj-core/src/test/java/co/rsk/rpc/netty/Web3HttpServerTest.java

```diff
@@ -298,6 +306,229 @@ void smokeTestProducesMethodInvalidException() throws Exception {
         smokeTest(APPLICATION_JSON, "localhost", filteredModules, null, mockResult, "web3sha3");
     }
 
+    @Test
+    void testMaxResponseSize() throws Exception {
+        Web3 web3Mock = Mockito.mock(Web3.class);
+        String mockResult = getMediumJsonResponse();
+        int responseSize = mockResult.getBytes().length * 2 - 1;
```

Comment:
Taking the string as source is not a good estimation if it includes spaces, new lines, etc. If you get the JsonNode from it and then do `writeValueAsBytes` you will get a size of `3655` vs `mockResult.getBytes().length` which is `4610`, not a negligible difference.